xml_converter/src/parseable.cpp

```diff
@@ -38,7 +38,9 @@ bool Parseable::init_xml_attribute(rapidxml::xml_attribute<>* attribute, vector<
 }
 
 vector<string> Parseable::as_xml() const{
+    cout << "error: Parseable::as_xml() should not be called";
+    exit(EXIT_FAILURE);
```

Comment:
Lets use exceptions instead of exit here. This is so we can catch them in a test case.